src/StripeTests/Entities/BankAccountTest.cs

```diff
@@ -8,31 +8,42 @@ namespace StripeTests
     using Stripe;
     using Xunit;
 
-    public class CustomerBankAccountTest : BaseStripeTest
+    public class BankAccountTest : BaseStripeTest
     {
         [Fact]
-        public void Deserialize()
+        public void DeserializeForAccount()
+        {
+            string json = GetFixture("/v1/accounts/acct_123/external_accounts/ba_123");
+            var bankAccount = Mapper<BankAccount>.MapFromJson(json);
+            Assert.NotNull(bankAccount);
+            Assert.IsType<BankAccount>(bankAccount);
+            Assert.NotNull(bankAccount.Id);
+            Assert.Equal("bank_account", bankAccount.Object);
+        }
+
+        [Fact]
+        public void DeserializeForCustomer()
         {
             string json = GetFixture("/v1/customers/cus_123/bank_accounts/ba_123");
-            var bankAccount = Mapper<CustomerBankAccount>.MapFromJson(json);
+            var bankAccount = Mapper<BankAccount>.MapFromJson(json);
             Assert.NotNull(bankAccount);
-            Assert.IsType<CustomerBankAccount>(bankAccount);
+            Assert.IsType<BankAccount>(bankAccount);
             Assert.NotNull(bankAccount.Id);
             Assert.Equal("bank_account", bankAccount.Object);
         }
 
         [Fact]
-        public void DeserializeWithExpansions()
+        public void DeserializeWithExpansionsForCustomer()
```

Comment:
Feel free to ignore for this PR, but `Account` should also be expandable on `BankAccount`.

Reply:
Ah yes that's because BankAccount is not customer specific anymore. FWIW I added the tests in `ExternalAccount` instead as I think it makes more sense in general. I also had to add support for `Account` on `BankAccount`.